outbound.js

```diff
@@ -187,6 +211,18 @@
         return deferred.promise;
     };
 
+    Outbound.prototype.trackBatch = function(events) {
```

Comment:
Alright, validation added in https://github.com/idris/lib-javascript/commit/eaea2f6c0fe3e0df9f623aae02d3de122723da94